code/modules/antagonists/disease/disease_abilities.dm

```diff
@@ -54,7 +54,7 @@ GLOBAL_LIST_INIT(disease_ability_singletons, list(
 			stage_speed += initial(S.stage_speed)
 			transmittable += initial(S.transmittable)
 			threshold_block += "<br><br>[initial(S.threshold_desc)]"
-		stat_block = "Resistance: [resistance]<br>Stealth: [stealth]<br>Stage Speed: [stage_speed]<br>Transmittability: [transmittable]<br><br>"
+		stat_block = "Resistance: [resistance]<br>Stealth: [stealth]<br>Stage Speed: [stage_speed]<br>Transmissibility: [transmittable]<br><br>"
```

Comment:
Is there a reason why this was changed or that the code thing kept the old change?

I can't find a reputable definition for either word and it's the same definition with what I can find.

Reply:
In the context of medical terminology, transmissible is preferred. Only corrected what was forward facing, since updating all the code for it would touch a -lot- of entries.